packages/integrations/mdx/src/plugins.ts

```diff
@@ -145,6 +146,12 @@ export function getRehypePlugins(mdxOptions: MdxOptions): MdxRollupPluginOptions
 		// computed from `astro.data.frontmatter` in VFile data
 		rehypeApplyFrontmatterExport,
 	];
+
+	if (mdxOptions.optimize) {
+		const options = mdxOptions.optimize === true ? undefined : mdxOptions.optimize;
```

Comment:
Why do we pass `undefined` here? Can you add a comment to the code? It looks "strange" from a beginner (me) point to view